### PR TITLE
Sb static pages/change mail widget

### PR DESCRIPTION
**Pull request by felixdausch**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/views/Widgets/Customer/ChangeMailWidget.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "identifier": "Ceres::ChangeMailWidget",
+    "label": "Widget.changeMailLabel",
+    "previewImageURL": "/images/widgets/address.svg",
```
Comment (Tim-M-S) [inline resources/views/Widgets/Customer/ChangeMailWidget.json]: change-mail.svg

File: src/Widgets/Presets/DefaultChangeMailPreset.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Ceres\Widgets\Presets;
+
+use Ceres\Widgets\Helper\Factories\PresetWidgetFactory;
+use Ceres\Widgets\Helper\PresetHelper;
+use Plenty\Modules\ShopBuilder\Contracts\ContentPreset;
+use Plenty\Plugin\Translation\Translator;
```
Comment (Tim-M-S) [inline src/Widgets/Presets/DefaultChangeMailPreset.php]: Wird nicht verwendet?

File: src/Widgets/Presets/DefaultChangeMailPreset.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?php
```
Comment (Tim-M-S) [inline src/Widgets/Presets/DefaultChangeMailPreset.php:1]: Fehlt noch in shopBuilder.json

File: resources/views/Widgets/Customer/ChangeMailWidget.twig [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+{% set inlineSpacings   = WidgetHelper.getInlineSpacings(spacingSettings, "", spacingSettings.customMargin.mobile) %}
+{% set spacingClasses   = WidgetHelper.getSpacingClasses(spacingSettings, "", spacingSettings.customMargin.mobile) %}
+{% set appearance       = widget.settings.appearance.mobile | default("primary") %}
+
+{{ Twig.set("contactId", "request.get(\"contactId\", 0)") }}
+{{ Twig.set("hash", "request.get(\"hash\", null)") }}
+{{ Twig.if("(contactId > 0 and hash is not null and not services.authentication.isLoggedIn()) or #{ isPreview | json_encode }") }}
+    {{ Twig.set("hashData", "services.userDataHash.getData(hash, contactId)") }}
+    {{ Twig.if("(hashData is not null and hashData.newMail is not null) or #{ isPreview | json_encode }") }}
+        <div class="widget widget-change-mail widget-{{ appearance }}{% if customClass | length > 0 %} {{ customClass }}{% endif %}{% if spacingClasses | length > 0 %} {{ spacingClasses }}{% endif %}"
```
Comment (Tim-M-S) [inline resources/views/Widgets/Customer/ChangeMailWidget.twig:15]: {{ appearance | default("primary") }}

**Review by Tim-M-S — CHANGES_REQUESTED** [review]

**Review by Tim-M-S — APPROVED** [review]

File: resources/lang/en/Widget.properties [hunk 5]
```diff
@@ -5,28 +5,31 @@ breadcrumbShowOnCheckoutLabel = "Display in checkout"
 breadcrumbShowOnContentCategoryLabel = "Display in content categories"
 breadcrumbShowOnHomepageLabel = "Display on homepage"
 breadcrumbShowOnMyAccountLabel = "Display in My account area"
+; Change mail
+changeMailLabel = "Change e-mail address"
```
Comment (fmutschler) [inline resources/lang/en/Widget.properties:9]: changeMailLabel = "Change email address"

File: resources/lang/en/Widget.properties [hunk 6]
```diff
@@ -5,28 +5,31 @@ breadcrumbShowOnCheckoutLabel = "Display in checkout"
...
+presetHeaderDefault = "Header"
+presetHomepageDefault = "Homepage"
+presetLegalDisclosureDefault = "Legal disclosure"
+presetLoginDefault = "Login page"
+presetMyAccountDefault = "My account"
+presetPrivacyPolicyDefault = "Privacy policy"
+presetSingleItemDefault = "Single item"
+presetPageNotFound = "Page not found"
+presetBasketDefault = "Basket"
+presetChangeMailDefault = "Change e-mail"
```
Comment (fmutschler) [inline resources/lang/en/Widget.properties:32]: presetChangeMailDefault = "Change email"

**Pull request merged**